### DISCUSSION TITLE
Access fluid state data from components after solver

### DISCUSSION THREAD

**Discussion by dpokras** [oemof/tespy#485] [Q&A]
I want to iterate through all the components in the network, access the fluid state property data (e.g. p, h, T, Q, v, s...) and keep them in a dictionary. Does anyone know of a straight forward method to do this? 

I imagine its something like iterating through nw.comps but this is where I get stuck.

**Comment by fwitte** [comment]
Hi @dpokras:

the information should be available from the connection results in a pandas dataframe:

```python
nw.results["Connection"]
```

Best

Francesco
